Makefile

```diff
@@ -14,6 +14,9 @@ imports:
 fmt:
 	@(gofmt -w .)
 
+lint:
+	@(go mod download && golangci-lint run)
```

Comment:
What's the purpose of `go mod download` here? Is this to install `golangci-lint`? If so, wouldn't we need to add it into `go.mod`?

Reply:
I faced the same follwing issue in travis-ci https://github.com/golangci/golangci-lint/issues/361
So I added `go mod download` before `golangci-lint run`